build-info-extractor/src/main/java/org/jfrog/build/extractor/ModuleParallelDeployHelper.java

```diff
@@ -44,8 +44,12 @@ private void deploy(ArtifactoryManager artifactoryManager, Set<DeployDetails> de
         deployableArtifacts.forEach(artifact -> {
             try {
                 ArtifactoryUploadResponse response = artifactoryManager.upload(artifact, logPrefix);
+                // Save information returned from Artifactory after deployed.
                 artifact.setDeploySucceeded(true);
                 artifact.setSha256(response.getChecksums().getSha256());
+                // In case of a SNAPSHOT deploy artifact's name is being modify by artifactory.
```

Comment:
// In case of a SNAPSHOT deploy artifact's name is being modify by artifactory.
-->
// When a maven SNAPSHOT artifact is deployed, Artifactory adds a timestamp to the artifact name, after the artifact is deployed.